src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -174,6 +176,10 @@ private static String updateUser(Request req, Response res) {
         String userId = req.params("id");
         Auth0UserProfile user = getUserById(userId);
 
+        if (user == null) {
+            logMessageAndHalt(req, 404, "User not found");
```

Comment:
I feel like this message might be a little vague given the UserController context. Maybe it should clarify which user was not found (the user to be created vs. the creating/requesting user).

Reply:
Added more context to this message in a recent commit.